docs/release_notes/ibexa_dxp_v4.5.md

```diff
@@ -0,0 +1,215 @@
+---
+description: Ibexa DXP v4.5 adds translation comparison, improvements to Customer Portal, Ibexa Commerce and Personalization.
+---
+
+# Ibexa DXP v4.5
+
+**Version number**: v4.5
+
+**Release date**: April 27, 2023
+
+**Release type**: [Fast Track](https://support.ibexa.co/Public/service-life)
+
+**Update**: [v4.4.x to v4.5](https://doc.ibexa.co/en/latest/update_and_migration/from_4.4/update_from_4.4/)
+
+## Notable changes
+
+### All-new Ibexa Commerce packages [[% include 'snippets/commerce_badge.md' %]]
+
+This release brings new packages to complement the redesigned and reconstructed Commerce offering.
+You can use them to further enhance your e-commerce presence:
+
+- `ibexa/order-management`
+- `ibexa/payment`
+
+Modules can interact with each other, for example, to decrease stock as a result of a sale, or cancel shipments and payments when orders are cancelled, and so on.
+
+#### Order management
+
+With order management in place, users can now search for orders and filter search results. 
+They can also review order details and completion status, as well as cancel orders.
+
+![The order list screen](img/4.5_order_list.png "The order list screen")
+
+#### Payment
+
+With the Payment module, users can define and manage payment methods, 
+as well as create and manage payments, search for payments and filter the results. 
+Users can also enable or disable payment methods, modify payment information, and cancel payments.
+
+![The payment methods screen](img/4.5_shipping_methods.png "The payment methods screen")
+
+For more information, see [Commerce](https://doc.ibexa.co/en/4.4/commerce/commerce/).
+
+### Changes to the Checkout package
+
+The Checkout package has been modified by adding shipping management capabilities that let users
```

Comment:
Checkout also includes new payment methods.

Reply:
Checkout as a proces, but not package - it is in ibexa/payment